src/coreclr/src/jit/lclvars.cpp

```diff
@@ -2670,7 +2670,49 @@ void Compiler::lvaSetStruct(unsigned varNum, CORINFO_CLASS_HANDLE typeHnd, bool
         compGSReorderStackLayout = true;
         varDsc->lvIsUnsafeBuffer = true;
     }
+#ifdef DEBUG
+    if (doExtraSuperPmiQueries)
+    {
+        makeExtraStructQueries(typeHnd, 2);
```

Comment:
Presumably you could make `2` an option, but I guess it's fine to hard-code it.

Reply:
I thought about making it an option, but it didn't seem worth the complexity, albeit minor.